models/src/main/java/io/sphere/sdk/states/commands/updateactions/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * {@doc.gen summary categories}
```

Comment:
Yes, I fixed it in https://github.com/sphereio/sphere-jvm-sdk/pull/344/files#diff-3c8dac3fa3b8753175c3b556f474c098L2